apps/meteor/server/settings/setup-wizard.ts

```diff
@@ -1223,6 +1223,13 @@ export const createSetupWSettings = () =>
 				secret: true,
 			});
 
+			await this.add('Omnigateway_Url', 'https://omni-gateway.rocket.chat', {
```

Comment:
Do we need this setting? I know apps use omnigateway but we actually never allowed customers to view/override it. This would allow that.

Reply:
Hey, this setting is going to be really useful when we add a new staging omnigateway env. So the apps know to which env they should point out.
I don't think apps should be able to override it, only to read so we can centralize this piece of information in one place, wdyt?

Reply:
This is a new initiative to stage omnigateway btw

Reply:
oh, btw, since this is a hidden setting the app creator will have to specify which permissions does it require, and he needs to manually select each "hidden" they will need access to. So we don't give complete power to app's developer to mess with our internal settings